src/Maestro/Client/src/Generated/Builds.cs

```diff
@@ -239,7 +239,7 @@ internal async Task OnCreateFailed(HttpRequestMessage req, HttpResponseMessage r
             CancellationToken cancellationToken = default
         )
         {
-            if (body == default)
+            if (body == null)
```

Comment:
Is this related to https://github.com/dotnet/arcade/issues/3995 ?

Reply:
When we regenerate later, it will regen as `default(<type>)`, but for now this is okay (`default(refType)` is null). Want to check this in so I don't have to keep making fixes in my local branch

Reply:
>Want to check this in so I don't have to keep making fixes in my local branch

No kidding. This part of the change got me excited because of that.